inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.cpp

```diff
@@ -22,78 +25,86 @@ using DataType = ngraph::element::Type_t;
 using DataDims = ngraph::Shape;
 
 struct NonMaxSuppressionTestCase {
-    int64_t num_batches, num_boxes, num_classes, max_output_boxes_per_class;
-    float iou_threshold, score_threshold;
+    int64_t numBatches, numBoxes, numClasses, maxOutputBoxesPerClass;
+    float iouThreshold, scoreThreshold, softNMSSigma;
 };
 
-
 class DynamicToStaticShapeNonMaxSuppression : public CommonTestUtils::TestsCommon,
         public testing::WithParamInterface<std::tuple<DataType, DataType, NonMaxSuppressionTestCase>> {
 public:
     void SetUp() override {
         const auto& parameters = GetParam();
-        const auto& float_type = std::get<0>(parameters);
-        const auto& integer_type = std::get<1>(parameters);
-        const auto& nms_setup = std::get<2>(parameters);
+        const auto& floatType = std::get<0>(parameters);
+        const auto& integerType = std::get<1>(parameters);
```

Comment:
Nitpick: I guess intention here is not to pass float and integer types, but types for first two inputs here.

Reply:
I got your idea, but the floatType is used in 1st, 2nd, 4th, 5th, and 6th inputs, while integer is used in 3rd input.